clusterloader2/pkg/measurement/common/network/performance-measurement-helper.go

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package network
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+func (npm *networkPerformanceMeasurement) validate(config *measurement.Config) error {
+	var err error
+
+	if npm.testDuration, err = util.GetInt(config.Params, "duration"); err != nil {
+		return err
+	}
+
+	if npm.podRatio, err = util.GetString(config.Params, "ratio"); err != nil {
```

Comment:
I guess you missed the comment I added here. I think passing a ratio isn't the way to go.
Instead you should have two separate parameters:
- numberOfClients
- numberOfServers

This would make all the validation, parsing etc. much easier.

Reply:
Modified as per suggestion.